lib/auth/webauthncli/api.go

```diff
@@ -57,7 +57,13 @@ type LoginPrompt interface {
 	// PromptTouch prompts the user for a security key touch.
 	// In certain situations multiple touches may be required (PIN-protected
 	// devices, passwordless flows, etc).
-	PromptTouch()
+	//
+	// Error can always be nil depending on who is implementing
+	// this interface e.g:
+	//  - implementation by 'DefaultPrompt' (auth/webauthn/prompt) will never return an error.
+	//  - implementation by 'pwdlessLoginPrompt' (teleterm/clusters/cluster_auth) will return
+	//    stream errors if any.
```

Comment:
I think I would ditch this comment for a couple reasons:

1. If there's an error we want to handle it regardless.
2. Docs about the implementations live in the implementation, not the interface.